src/util/Constants.js

```diff
@@ -205,6 +205,7 @@ const Endpoints = exports.Endpoints = {
     return {
       Emoji: emojiID => `${root}/emojis/${emojiID}.png`,
       Asset: name => `${root}/assets/${name}`,
+      DefaultAvatar: number => `${root}/embed/avatars/${number}.png`,
```

Comment:
well you are passing an integer 0-5 instead of the original links, so this is broken then.